I also had to run:

> sudo chown -R 501:20 "/Users/<myusername>/.npm" 
> sudo npm install punycode --save 
> chown <myusername> package-lock.json  


`yarn start` was working for me, even though my `yarn install` was failing before the fixes listed in this PR. But after running the updated commands, `yarn install` exits with the error:

> error Error: EACCES: permission denied, unlink '/Users/<myusername>/Documents/owd/content/node_modules/istanbul-lib-report/node_modules/semver/LICENSE'

and `yarn start` fails with 

> Error: Cannot find module './metrics/envelope.js'

I am not sure if this is a me issue or a general issue, but thought I would share just in case it's not just me. 
Setting up on a brand new MBP with Apple M3 chip, which is causing issues everywhere, not just with `yarn` or MDN.

--- a/README.md
+++ b/README.md
@@ -308,22 +308,16 @@ name causing the problem.
 
 1. To fix problems with `gifsicle`:
 
-   ```shell
    brew install automake autoconf libtool
-   ```
 
 1. To fix problems with `pngquant-bin`:
 
-   ```shell
    brew install pkg-config
-   ```
 
 1. To fix problems with `mozjpeg`:
 
-   ```shell
-   brew install libpng
-   sudo ln -s /opt/homebrew/Cellar/libpng/1.6.40/lib/libpng16.a /usr/local/lib/libpng16.a
-   ```
+   brew install libpng sudo ln -s
+   /opt/homebrew/Cellar/libpng/1.6.40/lib/libpng16.a /usr/local/lib/libpng16.a
 
 You may need to adjust the path to `libpng16.a` depending on the version of
 `libpng` you have installed.

--- a/README.md
+++ b/README.md
@@ -41,37 +41,8 @@ app locally. Do this like so:
     cd yari
     yarn install
 
-<details>
-  <summary>Troubleshooting installation</summary>
-
-If you get errors while installing dependencies via yarn on a Mac, you may need
-to install some additional packages:
-
-1. First, install [brew](https://brew.sh/) if you haven’t already
-
-1. To fix problems with `gifsicle`:
-
-   ```shell
-   brew install automake autoconf libtool
-   ```
-
-1. To fix problems with `pngquant-bin`:
-
-   ```shell
-   brew install pkg-config
-   ```
-
-1. To fix problems with `mozjpeg`:
-
-   ```shell
-   brew install libpng
-   sudo ln -s /opt/homebrew/Cellar/libpng/1.6.40/lib/libpng16.a /usr/local/lib/libpng16.a
-   ```
-
-You may need to adjust the path to `libpng16.a` depending on the version of
-libpng you have installed.
-
-</details>
+See the [troubleshooting](#troubleshooting) section below if you run into
+problems.
 
 Now copy the `.env-dist` file to `.env`:
 
@@ -326,3 +297,33 @@ The default server port `:5042` might be in use by another process. To resolve
 this, you can pick any unused port (e.g., 6000) and run the following:
 
     echo SERVER_PORT=6000 >> .env
+
+### Yarn install errors
+
+If you get errors while installing dependencies via yarn on a Mac, you may need
+to install some additional packages. Check the error message for the package
+name causing the problem.
+
+1. First, install [brew](https://brew.sh/) if you haven’t already
+
+1. To fix problems with `gifsicle`:
+
+   ```shell
+   brew install automake autoconf libtool
+   ```
+
+1. To fix problems with `pngquant-bin`:
+
+   ```shell
+   brew install pkg-config
+   ```
+
+1. To fix problems with `mozjpeg`:
+
+   ```shell
+   brew install libpng
+   sudo ln -s /opt/homebrew/Cellar/libpng/1.6.40/lib/libpng16.a /usr/local/lib/libpng16.a
+   ```
+
+You may need to adjust the path to `libpng16.a` depending on the version of
+`libpng` you have installed.

